pkg/minikube/constants/constants.go

```diff
@@ -298,8 +298,8 @@ func GetKubeadmCachedImages(imageRepository string, kubernetesVersionStr string)
 			imageRepository + "k8s-dns-kube-dns" + ArchTag(true) + "1.14.13",
 			imageRepository + "k8s-dns-dnsmasq-nanny" + ArchTag(true) + "1.14.13",
 			imageRepository + "k8s-dns-sidecar" + ArchTag(true) + "1.14.13",
-			imageRepository + "etcd" + ArchTag(false) + "3.3.10",
-			imageRepository + "coredns" + ArchTag(false) + "1.3.1",
+			imageRepository + "etcd" + ArchTag(false) + "3.3.15-0",
```

Comment:
This will need some sort of "if v1_16_plus()` condition, so that we aren't accidentally downloading these newer versions for Kubernetes v1.14 users.

Reply:
sure